src/qstls.cpp

```diff
@@ -190,11 +190,11 @@ void Qstls::initialGuessAdr(const vector<double> &wvg_,
 // Compute auxiliary density response
 void Qstls::computeAdr() {
   const int nx = wvg.size();
-  if (in.getCoupling() == 0) {
-    fill(slfc, 0.0);
-    adr.fill(0.0);
-    return;
-  }
+  // if (in.getCoupling() == 0) {
```

Comment:
Remove dead code. Just make sure that we are solving the scheme correctly for zero coupling

Reply:
Tested for both qVS and qSTLS for rs=0 and got the HF ssf a result. So I think it's correct.